users/firewall.rst

```diff
@@ -45,11 +45,12 @@ Local Firewall
 --------------
 
 If your PC has a local firewall, you will need to open the following ports for
-incoming and outgoing traffic:
+incoming and outgoing traffic, assuming that you configured port **22000** in
```

Comment:
Since this port isn't visible in the config by default, maybe hedge this like the previous mention ("unless you changed it from the default" or something)?

Reply:
I moved it to a note